tsl/test/expected/cagg_api.out

```diff
@@ -0,0 +1,101 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+SET timezone TO PST8PDT;
+CREATE TABLE hyper_ts (
+  time timestamp NOT NULL,
+  value float
+);
+CREATE TABLE hyper_tstz (
+  time timestamptz NOT NULL,
+  value float
+);
+SELECT * FROM create_hypertable('hyper_ts', 'time');
+WARNING:  column type "timestamp without time zone" used for "time" does not follow best practices
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             1 | public      | hyper_ts   | t
+(1 row)
+
+SELECT * FROM create_hypertable('hyper_tstz', 'time');
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             2 | public      | hyper_tstz | t
+(1 row)
+
+INSERT INTO hyper_ts
+  SELECT time, ceil(random() * 100)::int
+    FROM generate_series('2025-01-01 0:00:00'::timestamp,
+                         '2025-01-01 23:59:59', '1m') time;
+INSERT INTO hyper_tstz
+  SELECT time, ceil(random() * 100)::int
+    FROM generate_series('2025-01-01 0:00:00'::timestamptz,
+                         '2025-01-01 23:59:59', '1m') time;
+CREATE MATERIALIZED VIEW ts_temperature_1h
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('1 hour', time),
+       avg(value)
+  FROM hyper_ts
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "ts_temperature_1h"
+CREATE MATERIALIZED VIEW ts_temperature_15m
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('15 minutes', time),
+       avg(value)
+  FROM hyper_ts
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "ts_temperature_15m"
+CREATE MATERIALIZED VIEW tstz_temperature_1h
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('1 hour', time),
+       avg(value)
+  FROM hyper_tstz
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "tstz_temperature_1h"
+CREATE MATERIALIZED VIEW tstz_temperature_15m
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('15 minutes', time),
+       avg(value)
+  FROM hyper_tstz
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "tstz_temperature_15m"
+-- Generate some invalidations
+INSERT INTO hyper_ts
+  SELECT time, ceil(random() * 100)::int
+    FROM generate_series('2025-01-02 10:00:00'::timestamp,
+                         '2025-01-02 11:12:+02', '9 minutes') time;
+INSERT INTO hyper_tstz
+  SELECT time, ceil(random() * 100)::int
+    FROM generate_series('2025-01-02 12:00:00'::timestamptz,
+                         '2025-01-02 13:12:+02', '8 minutes') time;
+-- Check that there indeed is something in the hypertable invalidation
+-- log. If not, this will fail anyway.
+SELECT * FROM _timescaledb_catalog.continuous_aggs_hypertable_invalidation_log;
+ hypertable_id | lowest_modified_value | greatest_modified_value 
+---------------+-----------------------+-------------------------
+(0 rows)
```

Comment:
For some reason, no invalidations are added to the invalidation log.

Reply:
This can occur if the invalidation threshold is before the inserted rows. Test is updated to not do this.